### ISSUE TITLE
update() faster than second marxan()

### ISSUE THREAD

**Issue by JavierFajado** [paleo13/marxan#21]
Hi!

I have noticed that update() a MarxanSolved with new parameters is much faster on my computer than performing a new marxan() execution with those same new parameters.
As (I think) this update does not have an equivalent on running Marxan unconnected form R, I am wondering if there is any difference between those two ways, and if one is more appropriate than the other for any reason.
Thanks for the help! (and the package)

**Comment by jeffreyhanson** [comment]
Yeah, if you're using shapefiles as planning units if you use the update function this should be faster since it doesn't have to make the boundary data file again. So, using update would be better then creating a whole new problem each time.
